public/css/profile.css

```diff
@@ -95,3 +95,145 @@ body {
 #accessKeySubmitButton {
   margin-top: 15px
 }
+
+#profilePicDiv {
+  display: inline-block;
+  float: left;
+}
+
+#profilePicImg {
+  width: 200px;
+  height: 200px;
+  border-radius: 5px;
+  -webkit-border-radius: 5px;
+  -moz-border-radius: 5px;
+  box-shadow: 0 0 8px rgba(0, 0, 0, .8);
+  -webkit-box-shadow: 0 0 8px rgba(0, 0, 0, .8);
+  -moz-box-shadow: 0 0 8px rgba(0, 0, 0, .8);
+}
+
+#profileInfo {
+  display: inline-block;
+  margin-left: 50px
+}
+.wrapper {
+  margin: 0 auto;
+  padding: 20px 0px 5px 40px;
+  max-width: 1000px;
+  margin-left: 300px;
+}
+
+.table {
+  margin: 0 0 40px 0;
+  width: 100%;
+  box-shadow: 0 1px 3px rgba(0, 0, 0, 0.2);
+  display: table;
+}
+@media screen and (max-width: 580px) {
+  .table {
+    display: block;
+  }
+}
+
+.row {
```

Comment:
you should make sure all the common css rules across your site are in `style.css` and you include that in every page. Then you can have somethings like `profile.css` that has **only the specific styles for profile.html**